worker-markup-core/src/main/java/com/github/cafdataprocessing/worker/markup/core/EmailSplitter.java

```diff
@@ -132,8 +129,17 @@ public void generateEmailTags(Document doc) throws JDOMException, ExecutionExcep
 
     private List<String> getSeparatedEmails(String emailContent) throws JDOMException, ExecutionException, InterruptedException
     {
-        List<Integer> indexes = jepExec.getMessageIndexes(emailContent);
-        return separatedIndividualMessages(emailContent, indexes);
+        List<Integer> emailStartLineNumbers = new ArrayList<>();
+        final String lineMarkers = TalonEmailSplitter.splitEmails(emailContent);
+        char[] markers = ((String) lineMarkers).toCharArray();
```

Comment:
No need to cast `lineMarkers` to a `String`, since it already is one.
```suggestion
        final char[] markers = lineMarkers.toCharArray();
```